curriculum/challenges/english/25-front-end-development/lecture-understanding-functional-programming/6734061fe116df617a564a37.md

```diff
@@ -2,13 +2,15 @@
 id: 6734061fe116df617a564a37
 title: What Is Currying, and How Does It Work?
 challengeType: 19
-# videoId: nVAaxZ34khk
+videoId: IA_nMbD2Nw
```

Comment:
while reviewing the video found a couple of graphic errors. Waiting for the new video and then I will replace the video id